esp/esdllib/esdl_script.cpp

```diff
@@ -1407,7 +1409,7 @@ void createEsdlTransformOperations(IArrayOf<IEsdlTransformOperation> &operations
     }
 }
 
-IEsdlTransformOperation *createEsdlTransformOperation(XmlPullParser &xpp, const StringBuffer &prefix, bool withVariables)
+IEsdlTransformOperation *createEsdlTransformOperation(XmlPullParser &xpp, const StringBuffer &prefix, bool withVariables, bool ignoreCodingErrors)
```

Comment:
The parameter has to be there because the function has to match the signature "esdlOperationsFactory_t" which allows functions to make use of the parameter, but doesn't require it.  It would generally not be passed to a given operation since they have their own ignore option.  Rather if the function itself is being called as a child and has an error in it's own code, it needs to know what to do.